dashboard/src/components/Config/AppRepoList/AppRepoRefreshAllButton.tsx

```diff
@@ -0,0 +1,31 @@
+import * as React from "react";
+
+import "./AppRepo.css";
+
+interface IAppRepoRefreshAllButtonProps {
+  error?: Error;
```

Comment:
you are not using the `error` nor the `kubeappsNamespace` properties, you can skip those.

Reply:
Yes, I will remove it.